saos-import/src/main/java/pl/edu/icm/saos/importer/commoncourt/judgment/process/CcjProcessingService.java

```diff
@@ -46,6 +52,11 @@
         
         if (oldJudgment != null) {
             judgmentOverwriter.overwriteJudgment(oldJudgment, ccJudgment, judgmentWithCorrectionList.getCorrectionList());
+            
+            List<EnrichmentTag> judgmentEnrichmentTags = enrichmentTagRepository.findAllByJudgmentId(oldJudgment.getId());
+            
+            enrichmentTagRepository.delete(judgmentEnrichmentTags);
```

Comment:
Wouldn't it be better to have a special deleting repository method for this? For example: enrichmentTagRepository.deleteAllByJudgmentId(int judgmentId) something similar to JudgmentCorrectionRepository.delete...